keyboards/kprepublic/bm40hsrgb/keymaps/skewwhiffy/.gitignore

```diff
@@ -0,0 +1 @@
+!keyboard-layout.png
```

Comment:
Did you intend for this image to be included or not? If yes, it needs to be rehosted on Imgur and displayed in `readme.md` using that link.

Reply:
Unfortunately, I cannot register on Imgur: it's blocked for new registrations in the UK atm.

Using postimage, hope that's okay.